nncf/torch/quantization/quantize_functions.py

```diff
@@ -136,7 +136,9 @@ def _quantize_autograd_to_range(input_, input_low, input_high, levels):
 # pylint:disable=abstract-method
 class ExportQuantizeToFakeQuantize(torch.autograd.Function):
```

Comment:
Could you rename `ExportQuantizeToFakeQuantize` -> `ExportQuantizeToFakeQuantizeV2` because changes is significant?

Reply:
I'm not sure that it's necessary, changes are not significant, no change output of export to onnx, just add support if toch.jit.trace.

Reply:
Ok, let's do it in the follow up PR if it will need.